system-parachains/bridge-hubs/bridge-hub-polkadot/primitives/src/lib.rs

```diff
@@ -19,6 +19,9 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+extern crate alloc;
+
+use alloc as sp_std;
```

Comment:
also here, but we can also do in another PR

Reply:
Yes. We need to open a fix PR in PSDK.